crates/sui-graphql-rpc/src/types/transaction_block_kind/randomness_state_update.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    context_data::db_data_provider::PgManager,
+    types::{base64::Base64, epoch::Epoch},
+};
+use async_graphql::*;
+use sui_types::transaction::RandomnessStateUpdate as NativeRandomnessStateUpdate;
+
+#[derive(Clone, Eq, PartialEq)]
+pub(crate) struct RandomnessStateUpdateTransaction(pub NativeRandomnessStateUpdate);
+
+#[Object]
```

Comment:
Maybe I missed its definition, but some description of what a RandomnessState means would be useful.

Reply:
It's a new (work in progress) feature, to support native on-chain randomness.  I will add a note explaining what this transaction does here.  There will also be a follow-up PR to generally improve the documentation of all the transaction kinds.

Reply:
Sounds good, thanks!